osu.Game/Rulesets/Mods/ModMuted.cs

```diff
@@ -64,6 +66,11 @@ public abstract class ModMuted<TObject> : ModMuted, IApplicableToDrawableRuleset
             Value = true
         };
 
+        protected ModMuted()
+        {
+            InverseMuting.BindValueChanged(i => MuteComboCount.MinValue = i.NewValue ? 1 : 0, true);
```

Comment:
I'm not convinced that this is going to work 100% correctly with `CopyFrom()`. Whether it will depends on the order of property binding I believe.

Consider:
```csharp
var mod = new MutedMod { InverseMuting = { Value = true }, MuteComboCount = { Value = 100 } };
mod.CopyFrom(new MutedMod { InverseMuting = { Value = false }, MuteComboCount = { Value = 0 } };
```

If `CopyFrom` transfers `InverseMuting` first then the copy will work correctly. If it doesn't, it won't. This needs to be unit tested.

Reply:
It actually doesn't matter, since it binds the setting to the target setting and then unbinds from it, copying both the value and the defined limits.

I'll write a unit test for this as it could unintentionally break.